frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/FieldRow/FieldRowContainer.tsx

```diff
@@ -347,52 +296,14 @@ const FieldRowContainer = ({
                 </FormProvider>
               </Box>
               <Collapse in={isActive} style={{ width: '100%' }}>
-                <Flex
-                  px={{ base: '0.75rem', md: '1.5rem' }}
-                  flex={1}
-                  borderTop="1px solid var(--chakra-colors-neutral-300)"
-                  justify="flex-end"
-                >
-                  <ButtonGroup
-                    variant="clear"
-                    colorScheme="secondary"
-                    spacing={0}
-                  >
-                    {isMobile ? (
-                      <IconButton
-                        variant="clear"
-                        colorScheme="secondary"
-                        aria-label="Edit field"
-                        icon={<BiCog fontSize="1.25rem" />}
-                        onClick={handleEditFieldClick}
-                      />
-                    ) : null}
-                    {
-                      // Fields which are not yet created cannot be duplicated
-                      fieldBuilderState !== FieldBuilderState.CreatingField && (
-                        <Tooltip label="Duplicate field">
-                          <IconButton
-                            aria-label="Duplicate field"
-                            isDisabled={isAnyMutationLoading}
-                            onClick={handleDuplicateClick}
-                            isLoading={duplicateFieldMutation.isLoading}
-                            icon={<BiDuplicate fontSize="1.25rem" />}
-                          />
-                        </Tooltip>
-                      )
-                    }
-                    <Tooltip label="Delete field">
-                      <IconButton
-                        colorScheme="danger"
-                        aria-label="Delete field"
-                        icon={<BiTrash fontSize="1.25rem" />}
-                        onClick={handleDeleteClick}
-                        isLoading={deleteFieldMutation.isLoading}
-                        isDisabled={isAnyMutationLoading}
-                      />
-                    </Tooltip>
-                  </ButtonGroup>
-                </Flex>
+                {isActive && (
+                  <FieldButtonGroup
```

Comment:
yes! because various hooks such as `useCreateTabForm` `useDuplicateFormField` `useDeleteFormField` can now be called only in the new `FieldButtonGroup` component. Thus when these hooks changes, it will only cause re-render in the `FieldButtonGroup`

<img width="1791" alt="image" src="https://github.com/opengovsg/FormSG/assets/59867455/9cf84d69-a975-4ab7-b0b3-5a600a7a31ea">

Before:
(Hooks causing re-renders for all field rows)
<img width="1791" alt="image" src="https://github.com/opengovsg/FormSG/assets/59867455/177f7e59-0426-42fa-9dc3-4b82712d4b03">

Reply:
Thanks for the explanation @foochifa 🙏 🌮